mele-manoa/mele-manoa
# Issue 10 - Clone meteor application and cleanup

Steps to test
- Go to branch issue-10
- Run the application using meteor npm install and meteor npm run start and make sure the application doesn't crash or get any errors
- Make sure there are not eslint errors with meteor npm run lint
